libs/common/src/sql-db-connection/migrations/1734616272101-AddFeeRecordCorrectionTransientFormDataTable.ts

```diff
@@ -0,0 +1,26 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddFeeRecordCorrectionTransientFormDataTable1734616272101 implements MigrationInterface {
```

Comment:
could we leave the number out of the class?

Reply:
This is automatically generated by typeorm and matches the number in the file name. I don't think we should change this.